verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -0,0 +1,262 @@
+        required: true
+        schema:
+          type: string
+          pattern: '^[a-f\d]{24}$'
+    get:
+      summary: Gets a specific source
+      operationId: getSource
+      responses:
+        "200":
+          $ref: "#/components/responses/200Source"
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+        "422":
+          $ref: "#/components/responses/422"
+        "500":
+          $ref: "#/components/responses/500"
+    put:
+      summary: Updates a specific source
+      operationId: updateSource
+      requestBody:
+        description: Source to update
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Source"
+      responses:
+        "200":
+          $ref: "#/components/responses/200Source"
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+        "422":
+          $ref: "#/components/responses/422"
+        "500":
+          $ref: "#/components/responses/500"
+    delete:
+      summary: Deletes a specific source
+      operationId: deleteSource
+      responses:
+        "204":
+          description: Source deleted
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+        "500":
+          $ref: "#/components/responses/500"
+  /users:
+    get:
+      summary: Lists users
+      operationId: listUsers
+      parameters:
+        - name: page
+          in: query
+          description: The pages of users to skip before starting to collect the result set
+          required: false
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            default: 1
+        - name: limit
+          in: query
+          description: The number of users to return
+          required: false
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 100
+            default: 10
+      responses:
+        "200":
+          $ref: "#/components/responses/200UserArray"
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+        "422":
+          $ref: "#/components/responses/422"
+        "500":
+          $ref: "#/components/responses/500"
+  /users/{id}:
+    parameters:
+      - name: id
+        in: path
+        description: The user ID
+        required: true
+        schema:
+          type: string
+          pattern: '^[a-f\d]{24}$'
+    put:
+      summary: Updates a specific user's roles
+      operationId: updateUser
+      requestBody:
```

Comment:
it looks like the body of this request is currently a list of roles, not the whole user obj: https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/verification/curator-service/api/src/controllers/users.ts#L58

Reply:
Ah yea I was thinking it's possible to pass the whole user object, and we'll only access the roles field. But it's worth making it explicit that roles is the only field we're using. Done.